content/chapitres/introduction-to-git.adoc

```diff
@@ -258,7 +258,7 @@ image::scm-merge.png[caption="SCM Merge"]
 [source,bash]
 
 --
-git checkout master
+git switch master
```

Comment:
(Suggestion, non blocking): should we consider using main instead of master?  

```suggestion
git switch main
```

If yes, maybe let's do this in another PR :)

Reply:
good catch.